client/cli/cli.go

```diff
@@ -87,6 +87,31 @@ func Run(args []string) {
 	}()
 
 	cmds := command.DebugCommands()
+	names := cmds.Names()
+
+	// set command completer
+	t.line.SetWordCompleter(func(line string, pos int) (head string, completions []string, tail string) {
+		start := strings.LastIndex(line[:pos], " ")
+		match := ""
+		if start < 0 { // this is the command to match
+			head = ""
+			tail = line[pos:]
+			match = line
+		} else if strings.HasPrefix(line, "help ") {
```

Comment:
Currently `help` takes no arguments, so maybe hold off on this until that is implemented.